candle-onnx/tests/ops.rs

```diff
@@ -1639,3 +1773,450 @@ fn test_reduce_mean() -> Result<()> {
 
     Ok(())
 }
+
+// "Sqrt"
+#[test]
+fn test_sqrt() -> Result<()> {
+    // https://github.com/onnx/onnx/blob/main/docs/Operators.md#examples-155
+    test(&[1., 4., 9.], &[1., 2., 3.])?;
+
+    fn test(data: impl NdArray, expected: impl NdArray) -> Result<()> {
```

Comment:
I've followed the code pattern present in other tests, but it gets a bit repetitive, I think that a refactor using a builder pattern would be really great for all this tests, something like:
```rust
OnnxTest::build()
  .input(&[1, 2, 3, 4, ...])
  .input(1i64)
  .attribute("axis", 1)
  .attribute("keepdims", false)
  .expect(&[[1, 0, 0, 1, 0]])?
```
I've tried to be very consistent in the test code in order to streamline future refactors that you might want to do.

Reply:
Agreed that it's a bit repetitive but if it's in the tests that doesn't look too bad, feel free to make a separate PR to clean this out at some point if you want though.